src/features/content-insights/ContentInsightsSummary.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import ContentInsightsSummaryGhostState from './ContentInsightsSummaryGhostState';
+import GraphCardPreviewsSummary from './GraphCardPreviewsSummary';
+import OpenContentInsightsButton from './OpenContentInsightsButton';
+import { GraphData } from './types';
+
+import './ContentInsightsSummary.scss';
+
+interface Props {
+    graphData: GraphData;
+    isLoading: boolean;
+    onCtaClick: () => void;
```

Comment:
minor: could we rename this to just onClick since there's no other on click functions in this component? (unless this is the pattern we want to follow for naming onClick props)